ReactNativeClient/lib/BaseApplication.js

```diff
@@ -450,11 +450,16 @@ class BaseApplication {
 			refreshFolders = true;
 		}
 
-		if (this.hasGui() && ((action.type == 'SETTING_UPDATE_ONE' && action.key.indexOf('folders.sortOrder') === 0) || action.type == 'SETTING_UPDATE_ALL')) {
```

Comment:
Why this change?

Reply:
I did not see there was already a section for includeCompletedTodos, so I added one. I then notice that the check for action.type == 'SETTING_UPDATE_ALL' was duplicated in two sections so I combined into one.  I now notice that the original change was unneeded but also that the SETTING_UPDATE_ALL check is duplicated in quite a few other places as well.  In lieu of refactoring all of those I will just revert this file.

Reply:
Looks like I was mistaken again, the instances above are for refreshNotes not refreshFolders.  Back to my original point - I made the change because the check for SETTING_UPDATE_ALL was duplicated twice (now 3 times) and it seemed better to separate it.